### PR TITLE
Import twitter library only when it is used

### PR DESCRIPTION
**Pull request by oplantalech**

### CODE REVIEW

File: src/shared/components/rightbar/RightBar.tsx [hunk 1]
```diff
@@ -69,6 +68,7 @@ export default class RightBar extends React.Component<IRightBarProps, IRightBarS
                 </div>
             );
         } else {
+            let Timeline = require('react-twitter-widgets').Timeline;
```
Comment (alisman) [inline src/shared/components/rightbar/RightBar.tsx]: are you sure this works?   code importing should happen regardless

**Comment by jjgao** [comment]
Maybe we should just remove twitter as a whole? @alisman

**Comment by Sjoerd-van-Hagen** [comment]
@jjgao don't you think the Twitter feed is useful?

**Comment by jjgao** [comment]
@Sjoerd-van-Hagen I thought so but no one used it after a few months. We have removed it.

**Comment by alisman** [comment]
This script is for the twitter feed, not the twitter post.

**Comment by jjgao** [comment]
oh.. sorry. We need twitter feed.

**Comment by Sjoerd-van-Hagen** [comment]
@alisman can you test if it works for you?

File: src/shared/components/rightbar/RightBar.tsx (hunk 1 above)
Comment (alisman) [reply src/shared/components/rightbar/RightBar.tsx]: it does seem to work.  very surprised!

**Pull request merged**